samples/java/agents/magic_8_ball_security/README.md

```diff
@@ -0,0 +1,117 @@
+# Magic 8-Ball Security Agent
+
+This sample agent responds to yes/no questions by consulting a Magic 8-Ball.
+
+This agent demonstrates how to secure an A2A server with Keycloak using bearer token authentication and it shows how to configure an A2A client to specify the token when
+sending requests. The agent is written using Quarkus LangChain4j and makes use of the
+[A2A Java](https://github.com/a2aproject/a2a-java) SDK.
+
+## Prerequisites
+
+- Java 17 or higher
+- Access to an LLM and API Key
```

Comment:
It seems to need Docker or Podman as well

Reply:
You should mention you are using devservices for KeyCloak and for podman you should at least point to this https://quarkus.io/blog/quarkus-devservices-testcontainers-podman/